pulsar/README.md

```diff
@@ -0,0 +1,84 @@
+# Pulsar Bindings
+This document defines how to describe Apache Pulsar specific information with AsyncAPI.
+
+<a name="version"></a>
+
+## Version
+
+Current version is `0.1.0`.
+
+<a name="server"></a>
+## Server Binding Object
+
+This object contains information about the server representation in Pulsar.
+
+##### Fixed Fields
+
+Field Name | Type | Description |
+---|:---:|:---:|
+`tenant` | String | The pulsar tenant. If omitted, "public" must be assumed. |
+`bindingVersion` | String | The version of this binding. If omitted, "latest" MUST be assumed. |
+
+This object MUST contain only the properties defined above.
+
+##### Example
+
+```yaml
+servers:
+  production:
+    bindings:
+      pulsar:
+        tenant: contoso
+        bindingVersion: '0.1.0'
+```
+
+<a name="channel"></a>
+## Channel Binding Object
+This object contains information about the channel representation in Pulsar
+
+##### Fixed Fields
+
+Field Name | Type | Description |
+---|:---:|:---:|
+`namespace` | String | The namespace, the channel is associated with. If omitted, "public" MUST be assumed. |
+`persistence` | String | persistence of the topic in Pulsar `persistent` or `non-persistent`. |
```

Comment:
Should it be required?

Reply:
Yes. I was unsure about how to set that up specifically and what the recommendation was, but essentially yes. This particular one cannot be defaulted.

Reply:
Awesome. So then I'd just mark it as required:

```suggestion
`persistence` | String | **Required**. Persistence of the topic in Pulsar. Its value MUST be either `persistent` or `non-persistent`. |
```

Reply:
Awesome. I'll go over them and explicitly write required/optional then. Thanks!